ttnn/cpp/ttnn/operations/bernoulli/bernoulli_pybind.cpp

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "bernoulli_pybind.hpp"
+
+#include <optional>
+
+#include "bernoulli.hpp"
+#include "pybind11/decorators.hpp"
+
+namespace ttnn::operations::bernoulli {
+void bind_bernoulli_operation(py::module &module) {
+    std::string doc =
+        R"doc(Bernoulli(input: Tensor, output: Optional[Tensor] = None, dtype: Optional[DataType] = None, memory_config: Optional[MemoryConfig] = None, compute_kernel_config: Optional[ComputeKernelConfig] = None) -> Tensor
```

Comment:
@yan-zaretskiy , can you please help to craft a very short guide on the proper doc format so that we can share with everyone?

@BuiChiTrung , please see TT-NN op like eltwise binary for a sample of how we currently write docs.

Reply:
I've updated the doc.